dFusion/dFusionSpec.md

```diff
@@ -79,10 +79,93 @@ To summarize, here is a list of state that is stored inside the smart contract:
 ## Batch Auction Workflow
 
 The trading workflow consists of the following sequential processes:
+
+0. Account opening, deposits & withdrawals
 1. On-Chain order collection
-2. Finding the batch price: optimization of batch trading surplus (off-chain)
-3. Verifying batch price and trade execution (zkSnark)
-4. Restart with step 1
+2. Transition function from sha to Pedersen hashes (zkSnark)
```

Comment:
Can we use step 2..4 from the original?

Reply:
Yes, although I wasn't aware that I had changed this. Could it be that I was working with an older document?>